modules/cvv/samples/cvv_demo.cpp

```diff
@@ -42,29 +42,21 @@ main(int argc, char** argv)
 {
   cv::Size* resolution = nullptr;
 
-  // parse options
-  const char* optstring = "hr:";
-  int opt;
-  while ((opt = getopt(argc, argv, optstring)) != -1) {
-    switch (opt) {
-    case 'h':
-      usage();
-      return 0;
-      break;
-    case 'r':
-      {
-        char dummych;
-        resolution = new cv::Size();
-        if (sscanf(optarg, "%d%c%d", &resolution->width, &dummych, &resolution->height) != 3) {
-          printf("%s not a valid resolution\n", optarg);
-          return 1;
-        }
-      }
-      break;
-    default: /* '?' */
-      usage();
-      return 2;
-    }
+  // parser keys
+  const char *keys =
+      "{ help h usage ? |    | show this message }"
+      "{ resolution r   |0x0| resolution to width and height in the format WxH }";
+  CommandLineParser parser(argc, argv, keys);
+  string res(parser.get<string>("resolution"));
+  if (parser.has("help") || res == "0x0") {
+    usage();
+    return 0;
+  }
+  char dummych;
+  resolution = new cv::Size();
```

Comment:
Looks like `resolution` isn't released

Reply:
@terfendail You might be right, but I have not introduced or modified `resolution` allocation/deallocation myself, and it is been there in the original code the exact way. Is it my responsibility to address ALL issues, or just to avoid introducing new ones?

All I care about in this PR is to fix Visual Studio issues, so that it can compile. Of course you can push another PR to do the release.

Reply:
"Resolution" parameter were optional in original sample and using of pointer had some sense. Currently this parameter is required, so pointer can be dropped completely.

BTW, Calling parse function (scanf) after CommandLineParser doesn't look very good. Perhaps we could split this into two standalone parameters: "w" and "h". What do you think?

Reply:
I see! Sorry for the confusion...